### ISSUE TITLE
Adicionar suporte a NuConta

### ISSUE THREAD

**Issue by andreroggeri** [andreroggeri/nubank-sync-ynab#10]
Para sincronizar as transações da NuConta + os rendimentos

**Comment by gutobenn** [comment]
@andreroggeri Em breve abrirei um PR com a sincronização de transações da NuConta (sem rendimentos). Já tenho uma versão local funcionando, só preciso dar uma polida.

**Comment by andreroggeri** [comment]
Show @gutobenn

Uma dúvida, vc tem usado essa aplicação? Eu tive problemas na minha conta, parece que a lib que faz o acesso do YNAB está disparando alguns mecanismos de segurança, e por conta disso eu desliguei o meu script até atualizar pra lib do YNAB que usa a API (daí não precisa de usuário e senha)

**Comment by gutobenn** [comment]
@andreroggeri Então, descobri ela ontem e passei um bom tempo tentando fazer o setup local. Tive vários problemas com versões de dependências e incompatibilidade com meu SO, então optei por executá-la no Docker. Agora tá rodando legal :)

O que aconteceu com você? O YNAB bloqueou sua conta? E por quanto tempo? E você agendou para rodar 1x por dia? Percebi que a conta é temporariamente bloqueada após algumas tentativas de login (chuto que 5 a cada alguns minutos), mesmo que elas sejam bem sucedidas. A minha conta bloqueou algumas vezes enquanto estava fazendo os ajustes para a NuConta. Ao tentar acessar o site ele me falou do prazo que levaria para desbloquear (30 minutos na última vez que vi).

Enfim, essa atualização pra outra lib do YNAB será ótima! :)

**Comment by andreroggeri** [comment]
Sim, nos  lugares que eu estava logado continuei acessando normal, mas em novos lugares eu tive que pedir pro suporte do nubank liberar o meu IP.

VOu fazer essa migração que vai facilitar muito a vida 🤓

**Comment by gutobenn** [comment]
@andreroggeri alguma novidade sobre essa migração para outra lib? 

Recentemente comecei a receber o erro "Uunrecoverable server error, sorry YNAB", era esse erro que aparecia pra ti também? Tô achando que podem ter desabilitado de vez o login com usuário e senha.

**Comment by andreroggeri** [comment]
Na real eu acabei pensando em criar um script "unificado" que fosse fácil de testar e de importar de outros bancos também.

Disso nasceu outro repo => https://github.com/andreroggeri/br-to-ynab que atualmente suporta Nubank (Cartão e Conta) + Bradesco (Cartão e Conta).

Esse script funciona mas ainda não está fácil de usar, preciso retomar esse desenvolvimento

**Comment by gutobenn** [comment]
@andreroggeri Massa! Acabei de configurar aqui, já estou usando :))

Tive dificuldades para montar o ambiente e executar, recebi alguns erros como `ModuleNotFoundError: No module named 'br_to_ynab.importers'`. No fim, como queria resolver isso logo para fazer alguma sincronização ainda antes do fim do mês, fui cortando pedaços de código que eu não precisava e funcionou. Mas agora meu código está bem diferente do repositório hehe

Então se você puder documentar no repositório ou me indicar como fazer o setup do modo adequado será ótimo. Aí quem sabe também consigo contribuir no desenvolvimento, já tenho algumas correções que fiz.
Tentei alguns comandos como `python setup.py develop` mas não resolveu.
